drivers/video/sunxi/sunxi_display.c

```diff
@@ -2426,6 +2529,13 @@ void *video_hw_init(void)
 			}
 		}
 	}
+    console_variant = env_get("DETECTED_VERSION");
```

Comment:
this is an example - ok it works, but what u have in mind to use this implementation for? I assume for debugging instead of printing to stdout?

Reply:
Debug, showing detected LCD panel, version, I know that we have done it by imagemagick, but this is another option. Generally is nice to have feature